docs/geedocs/answer/2880343.html

```diff
@@ -41,6 +41,6 @@
 <a name="top_of_file"></a>
 <p><img src="../art/common/googlelogo_color_260x88dp.png" width="130" height="44" alt="Google logo" /></p>
 <h1><a href="../index.html">Google Earth Enterprise Documentation Home</a> | Troubleshooting and error messages</h1><h2>GEE downloads to Mac</h2>
-<p>Macs that run OS X Mountain Lion do not yet list the Google Support Portal as an "identified developer," so they place Portal downloads in the trash as a quarantine measure. If you download EC or Earth plug-in from the Support Portal to a Mac running Mountain Lion with security set to "maximum", you'll need to move them into the appropriate directory after download. For more information, see <a href="http://support.apple.com/kb/HT5290" target="_blank">Apple support</a>.</p>
+<p>Macs that run OS X Mountain Lion may not list the OpenGEE team as an "identified developer," so they may place OpenGEE downloads in the trash as a quarantine measure. If you download EC or Earth plug-in from the Support Portal to a Mac running Mountain Lion with security set to "maximum", you may need to move them into the appropriate directory after download. For more information, see <a href="http://support.apple.com/kb/HT5290" target="_blank">Apple support</a>.</p>
```

Comment:
I don't know if this page is still relevant.  I'm going to just delete it and we can add instructions back if someone reports a problem.